Jasmine/Common/Models/Coordinate.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+/// Defines a coordinate system that is zero-indexed, where the top left corner has the coordinate
+/// row = 0, col = 0.
+struct Coordinate {
+
+    /// Defines the origin of the coordinate system, which is zero.
+    static let zero = Coordinate(row: 0, col: 0)
```

Comment:
Perhaps 'origin' would be a better name? Zero would already imply the origin must be at (0, 0)